fhttp/http_test.go

```diff
@@ -841,6 +842,58 @@ func TestPayloadSizeLarge(t *testing.T) {
 	}
 }
 
+func TestUUIDFastClient(t *testing.T) {
+	m, a := DynamicHTTPServer(false)
+	m.HandleFunc("/", ValidateUUIDPath)
+	url := fmt.Sprintf("http://localhost:%d/{uuid}", a.Port)
+	o := HTTPOptions{URL: url, DisableFastClient: false}
+	client, _ := NewClient(&o)
+	code, data, header := client.Fetch()
```

Comment:
one missing test though is that if you make a 2nd request you get a different UUID in the context of the httprunner (you could set num request to 10, connections to 1 or 2 and check you get all different results)

Reply:
Done.